### ISSUE TITLE
Unknown algorithm(s) found: webauthn-sk-ecdsa-sha2-nistp256@openssh.com

### ISSUE THREAD

**Issue by pedrohdz** [jtesta/ssh-audit#149]
```console
$ ssh-audit -c

# general
(gen) client IP: 127.0.0.1
(gen) banner: SSH-2.0-OpenSSH_9.0
(gen) software: OpenSSH 9.0
(gen) compression: enabled (zlib@openssh.com, zlib)

# key exchange algorithms
(kex) diffie-hellman-group14-sha256                -- [info] available since OpenSSH 7.3, Dropbear SSH 2016.73
(kex) diffie-hellman-group16-sha512                -- [info] available since OpenSSH 7.3, Dropbear SSH 2016.73
(kex) diffie-hellman-group18-sha512                -- [info] available since OpenSSH 7.3
(kex) diffie-hellman-group-exchange-sha256         -- [info] available since OpenSSH 4.4
(kex) curve25519-sha256                            -- [info] available since OpenSSH 7.4, Dropbear SSH 2018.76
(kex) curve25519-sha256@libssh.org                 -- [info] available since OpenSSH 6.5, Dropbear SSH 2013.62
(kex) ext-info-c

# host-key algorithms
(key) ssh-ed25519                                  -- [info] available since OpenSSH 6.5
(key) ssh-ed25519-cert-v01@openssh.com             -- [info] available since OpenSSH 6.5
(key) sk-ssh-ed25519@openssh.com                   -- [info] available since OpenSSH 8.2
(key) sk-ssh-ed25519-cert-v01@openssh.com          -- [info] available since OpenSSH 8.2
(key) rsa-sha2-256                                 -- [info] available since OpenSSH 7.2
(key) rsa-sha2-512                                 -- [info] available since OpenSSH 7.2
(key) webauthn-sk-ecdsa-sha2-nistp256@openssh.com  -- [warn] unknown algorithm
(key) rsa-sha2-256-cert-v01@openssh.com            -- [info] available since OpenSSH 7.8
(key) rsa-sha2-512-cert-v01@openssh.com            -- [info] available since OpenSSH 7.8

# encryption algorithms (ciphers)
(enc) aes128-ctr                                   -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
(enc) aes192-ctr                                   -- [info] available since OpenSSH 3.7
(enc) aes256-ctr                                   -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
(enc) aes128-gcm@openssh.com                       -- [info] available since OpenSSH 6.2
(enc) aes256-gcm@openssh.com                       -- [info] available since OpenSSH 6.2
(enc) chacha20-poly1305@openssh.com                -- [info] available since OpenSSH 6.5
                                                   `- [info] default cipher since OpenSSH 6.9.

# message authentication code algorithms
(mac) hmac-sha2-256-etm@openssh.com                -- [info] available since OpenSSH 6.2
(mac) hmac-sha2-512-etm@openssh.com                -- [info] available since OpenSSH 6.2
(mac) umac-128-etm@openssh.com                     -- [info] available since OpenSSH 6.2



!!! WARNING: unknown algorithm(s) found!: webauthn-sk-ecdsa-sha2-nistp256@openssh.com.  Please email the full output above to the maintainer (jtesta@positronsecurity.com), or create a Github issue at <https://github.com/jtesta/ssh-audit/issues>.
```

**Comment by jtesta** [comment]
@pedrohdz : Thanks for reporting this!  I checked in an update to the master branch to address this (it should flag it as using a weak curve since its based on NIST P-256).  Can you give it a try and see if it works for you?

**Comment by pedrohdz** [comment]
@jtesta 
Looking good!  Thank you for the quick turnaround!

```
$ ssh-audit -c
# general
(gen) client IP: 127.0.0.1
(gen) banner: SSH-2.0-OpenSSH_9.0
(gen) software: OpenSSH 9.0
(gen) compression: enabled (zlib@openssh.com, zlib)

# key exchange algorithms
(kex) curve25519-sha256                            -- [info] available since OpenSSH 7.4, Dropbear SSH 2018.76
(kex) curve25519-sha256@libssh.org                 -- [info] available since OpenSSH 6.5, Dropbear SSH 2013.62
(kex) diffie-hellman-group16-sha512                -- [info] available since OpenSSH 7.3, Dropbear SSH 2016.73
(kex) diffie-hellman-group18-sha512                -- [info] available since OpenSSH 7.3
(kex) diffie-hellman-group-exchange-sha256         -- [info] available since OpenSSH 4.4
(kex) ext-info-c

# host-key algorithms
(key) ssh-ed25519                                  -- [info] available since OpenSSH 6.5
(key) ssh-ed25519-cert-v01@openssh.com             -- [info] available since OpenSSH 6.5
(key) sk-ssh-ed25519@openssh.com                   -- [info] available since OpenSSH 8.2
(key) sk-ssh-ed25519-cert-v01@openssh.com          -- [info] available since OpenSSH 8.2
(key) rsa-sha2-256                                 -- [info] available since OpenSSH 7.2
(key) rsa-sha2-256-cert-v01@openssh.com            -- [info] available since OpenSSH 7.8
(key) rsa-sha2-512                                 -- [info] available since OpenSSH 7.2
(key) rsa-sha2-512-cert-v01@openssh.com            -- [info] available since OpenSSH 7.8
(key) webauthn-sk-ecdsa-sha2-nistp256@openssh.com  -- [fail] using weak elliptic curves
                                                   `- [info] available since OpenSSH 8.3

# encryption algorithms (ciphers)
(enc) chacha20-poly1305@openssh.com                -- [info] available since OpenSSH 6.5
                                                   `- [info] default cipher since OpenSSH 6.9.
(enc) aes256-gcm@openssh.com                       -- [info] available since OpenSSH 6.2
(enc) aes128-gcm@openssh.com                       -- [info] available since OpenSSH 6.2
(enc) aes256-ctr                                   -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52
(enc) aes192-ctr                                   -- [info] available since OpenSSH 3.7
(enc) aes128-ctr                                   -- [info] available since OpenSSH 3.7, Dropbear SSH 0.52

# message authentication code algorithms
(mac) hmac-sha2-256-etm@openssh.com                -- [info] available since OpenSSH 6.2
(mac) hmac-sha2-512-etm@openssh.com                -- [info] available since OpenSSH 6.2
(mac) umac-128-etm@openssh.com                     -- [info] available since OpenSSH 6.2

# algorithm recommendations (for OpenSSH 9.0)
(rec) -webauthn-sk-ecdsa-sha2-nistp256@openssh.com -- key algorithm to remove
(rec) +diffie-hellman-group14-sha256               -- kex algorithm to append
(rec) +sntrup761x25519-sha512@openssh.com          -- kex algorithm to append

# additional info
(nfo) For hardening guides on common OSes, please see: <https://www.ssh-audit.com/hardening_guides.html>
```
